### DISCUSSION TITLE
Purchase order notification

### DISCUSSION THREAD

**Discussion by SergeoLacruz** [inventree/InvenTree#4895] [Ideas]
Hi,
When I set the Responsible user in a purchase order, he gets a notification when the order is created. IMHO it makes more sense to send out the notification then the order is placed. 
Inventree does not do any ordering. There are other tools for that like SAP. Usually these tools are handled by someone else, the responsible one. i would like to inform this guy after I added all the parts using the place button. 
What do you think?

Michael

**Comment by SchrodingersGat** [comment]
@SergeoLacruz this makes sense - a new purchase order is empty, so it's not much to notify on.

**Comment by SergeoLacruz** [comment]
Thanks for picking this up.
